src/community/ogcapi/ogcapi-features/src/main/resources/org/geoserver/ogcapi/v1/features/openapi.yaml

```diff
@@ -1156,6 +1156,10 @@ components:
         text/html:
           schema:
             type: string
+    OpenAPI:
```

Comment:
Wondering... is it legit to only have a description without a schema? Maybe it is, dunno, otherwise I guess we can link an external, online schema for OpenAPI itself.